tests/test_combineer_karteringen.py

```diff
@@ -0,0 +1,280 @@
+import geopandas as gpd
+from shapely.geometry import Polygon
+
+from veg2hab.vegkartering import Kartering, VegTypeInfo, _combineer_twee_geodataframes
+
+
+def _is_equal(gdf1: gpd.GeoDataFrame, gdf2: gpd.GeoDataFrame) -> bool:
+    # Zijn alle niet-geometry kolommen gelijk?
+    if not gdf1.drop(columns="geometry").equals(gdf2.drop(columns="geometry")):
+        return False
+
+    # Zijn alle geometrieën gelijk?
+    # We doen dit geometrisch ipv met equals want met equals is driehoek ABC niet gelijk aan driehoek ACB
+    return gdf1.symmetric_difference(gdf2).is_empty.all()
+
+
+def test_geen_overlap():
+    """
```

Comment:
nice die plaatjes!

Reply:
kan copilot dit?

Reply:
![image](https://github.com/user-attachments/assets/1744de42-1385-4332-9429-0d95be0604b0)

Nee hahaha, is nog wat te lastig

Hij kon wel bij een van de simpele gevallen van mn plaatje naar de goeie coordinaten, maar dat werd later ook te lastig.

Reply:
![image](https://github.com/user-attachments/assets/186f348c-cbba-449a-b270-f1ab5b795cb6)

Poging 2 met meer context lukte ook niet